### PR TITLE
Fetch button

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
something with my remove filter isn´t working correctly, but the rest is working I think! thank u for the review :)

### CODE REVIEW

**Comment by Alexander-Schulte** [comment]
Hi, just added the "var" for the colors. Implementet the two filters before the "fetch(url)". I´ll try to split the function, so its more readable. But for now, everythink is working fine I think! thank u for ur review!

File: src/lib/API.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        img.alt = `$(character.name)`;
+
+        if (character.status === "Alive") {
+          section.style.backgroundColor = "var(--section-alive-color)";
+        } else if (character.status === "Dead") {
+          section.style.backgroundColor = "var(--section-dead-color)";
+        } else if (character.status === "unknown") {
+          section.style.backgroundColor = "var(--section-unknown-color)";
+        }
+
```
Comment (ManuelRemer) [inline src/lib/API.js]: Here you create inline-css dynamically. I am not sure but i guess it is good practice to add or toggle here different classes. You could create classes for each status which represent only the background-color and add them as second class.

File: src/lib/API.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        const section = document.createElement("section");
+        section.classList.add("character");
+        const fullName = document.createElement("h2");
+        fullName.classList.add("fullName");
+        fullName.textContent = `${character.name}`;
+        const img = document.createElement("img");
+        img.classList.add("img");
+        img.src = character.image;
+        img.alt = `$(character.name)`;
+
```
Comment (ManuelRemer) [inline src/lib/API.js]: Here is an advice i got as well:
to shorten your higher order function you could separate lines 29 to 53 and declare several functions for creating the html-elements. Then just call them in your second "then" and append the elements.

**Review by ManuelRemer — APPROVED** [review]
nice code, have you deployed yet, looking forward to see it live

File: src/lib/API.js (hunk 2 above)
Comment (ManuelRemer) [reply src/lib/API.js]: and as you mentioned, you already plan to do that. Sorry have read your comment just after review.

**Pull request merged**